.flake8

```diff
@@ -13,5 +13,5 @@
 # https://github.com/PyCQA/pycodestyle/issues/386
 # W504: Raised by flake8 even when it is followed
 ignore = D203, E124, E126, F403, F405, F811, E402, E129, W605, W504
-max-line-length = 160
+max-line-length = 300
```

Comment:
it might be more readable for the `launch_cmd`s to be multi-line using this syntax for constructing string literals over multiple lines without newlines.

```
x = ("hello "
     "world"
     "!"
    )

print(x)
```

Reply:
Done